src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1393,6 +1395,28 @@ export function registerTerminalActions() {
 			}
 		}
 	});
+
+	registerActiveInstanceAction({
```

Comment:
This is the perfect type of thing to add to a terminalContrib as I think it's completely self contained.

Reply:
not following - we're using `sharedWhenClause` here too, so I think these are in the right place already?